apps/api/src/app/subscribers/usecases/chat-oauth-callback/chat-oauth-callback.usecase.ts

```diff
@@ -96,10 +97,14 @@ export class ChatOauthCallback {
     command: ChatOauthCallbackCommand,
     integrationCredentials: ICredentialsDto
   ): Promise<string> {
-    const redirectUri =
+    let redirectUri =
       process.env.API_ROOT_URL +
       `/v1/subscribers/${command.subscriberId}/credentials/${command.providerId}/oauth/callback?environmentId=${command.environmentId}`;
```

Comment:
The only thing is maybe we could extract the builder of redirectUri to some shared file so both usecases could use it, my mistake not doing that but it is not a must at all.

Reply:
Maybe we could extract the builder of redirectUri to some shared file so both usecases could use it, my mistake not doing that but it is not a must at all.